Content.Server/Body/Systems/BodySystem.cs

```diff
@@ -136,22 +130,16 @@ public override HashSet<EntityUid> GibBody(EntityUid bodyId, bool gibOrgans = fa
 
         _audio.Play(body.GibSound, filter, coordinates, true, audio);
 
-        var containers = GetBodyContainers(bodyId, body: body).ToList();
-
-        foreach (var container in containers)
+        foreach (var entity in gibs)
         {
-            foreach (var entity in container.ContainedEntities)
+            if (deleteItems)
+            {
+                QueueDel(entity);
+            }
+            else
             {
-                if (deleteItems)
-                {
-                    QueueDel(entity);
-                }
-                else
-                {
-                    container.Remove(entity, EntityManager, force: true);
-                    SharedTransform.SetCoordinates(entity,coordinates);
-                    entity.RandomOffset(0.25f);
-                }
+                SharedTransform.SetCoordinates(entity, coordinates);
+                entity.RandomOffset(0.25f);
```

Comment:
Do this in one method not two please.

Reply:
That code was already like that. It just shows as new since I un-indented it.

Reply:
Resolved this by just taking out the SetCoordinates call, it seems to not be necessary anymore.